src/Services/QueuedJobService.php

```diff
@@ -519,6 +519,21 @@ public function checkJobHealth($queue = null)
             ]
         );
 
+        // Send broken job email notification
+        $subject = _t(__CLASS__ . '.BROKEN_JOBS', 'Broken job(s)');
+        $message = sprintf(_t(
+            __CLASS__ . '.BROKEN_JOBS_MSG',
+            implode(' ', [
+                'The following job(s) appear to be broken.',
+                'Log in to %s to see further details and take any necessary actions.',
```

Comment:
The other templates have the "log in to etc etc" part in the template. We should aim to have this template be similar to the existing templates.

If you really want to use i18n on it, you should use the template `<%t %>` and update the other templates to also use it. See https://docs.silverstripe.org/en/5/developer_guides/i18n/#usage-in-template-files for docs on how to use i18n in templates.

Reply:
I don't feel strongly about it, I've updated it to make it similar to the other emails